src/parser/multiline_block_quote.rs

```diff
@@ -0,0 +1,9 @@
+/// The metadata of a multiline blockquote.
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct NodeMultilineBlockQuote {
+    /// The length of the fence.
+    pub fence_length: usize,
+
+    /// ??? The indentation level of the code within the block.
```

Comment:
Indentation level of the fence itself, in order to ensure that items within are correctly interpreted (e.g. per the "Marker can be indented up to 3 spaces" example).

Reply:
Spaced on updating that comment, thanks!